.github/workflows/test-indicators-coverage.yml

```diff
@@ -1,6 +1,6 @@
 # For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
 
-name: Indicators
+name: Publish code coverage
```

Comment:
I think I like what you're doing better for naming these.  A challenge I've had with these, is that they display differently in the PR checks list than they do in the main Actions side-nav menu; making them difficult to interpret.

- In PR Checks: [workflow name] / [job name]
- In Actions menu: [workflow name] (only)

I'd been prioritizing how they look in PR Checks with taxonomy like:

- [workflow name: subject of action] / [job name: action type]
- example: Indicators / unit tests

However, when there's several job names for "Indicators" it is difficult to find them in the [workflow name]-only side-nav in the Actions menu.  What you have is better:

- shown in checks: Publish code coverage / code coverage
- shown in actions: Publish code coverage

Though, perhaps inverting what I was doing might make these less redundant.

- [workflow name] / [job name]
- example: Publish code coverage / indicators

Just a suggestion.  Having a unique workflow name like you're doing here will make it easier to find in Actions, and having subject as job name might also help keep subject matter (target of action) in the checks list?

Reply:
Also, given that this is really the only unit test run that occurs on push to main, maybe we should call it "Test with code coverage" or something that better depict that main was tested?  Minor, not really important.

Reply:
Good suggestion. let's go with it.
```
[workflow name] / [job name]
 - workflow name: unique workflow name that can explain what it does.
 - job name: subject matter (target of action)
```

Reply:
@DaveSkender I have a question. What is the `test-result.xml` generated from unit test used for?

Reply:
It’s used to publish unit test results (not code coverage).  For example, in the old build system:

![](https://github.com/user-attachments/assets/7fc6b29f-9aa3-438c-9660-d9f7708d8dfc)

In GitHub Actions, I’m doing [something similar](https://github.com/DaveSkender/Stock.Indicators/runs/28332816918) with [publishing](https://github.com/DaveSkender/Stock.Indicators/blob/e01e844c01ba31549cd10656aac4a5395b476666/.github/workflows/test-indicators.yml#L85-L91) the .NET test results.

![](https://github.com/user-attachments/assets/014923f3-0cce-42e8-be32-5cdb550ea123)

I hadn’t figured out how to publish these yet.

Reply:
@DaveSkender Aha, then we are not using it now. Do you think we need to publish the result? Because the test envs are variant, so I think the report from one of the envs does not have lots of meaning. Just my thought..!

Reply:
@DaveSkender Applied new taxonomy. [c9a40d8](https://github.com/facioquo/stock-indicators-python/pull/386/commits/c9a40d83cb0d985950ef6234ca5b58ac567b4ff8)
If you are thinking of better name, feel free to change anything please.

Reply:
> the report from one of the envs does not have lots of meaning

it’s not anything to do with environments, aside from a filter that would prevent generating and publishing 10+ identical test reports from each matrix instance.  publishing them is just a nicer way to see exactly which tests passed or failed, so you don’t have to scroll through 1000s of lines of log out put to see it.  with that said, is it seriously important — no.

> workflow names

what you have is 👍🏼